Example/TABObserverSet/Helpers/UIAlertController+ObserverSet.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UIAlertController+ObserverSet.swift
+//  TABObserverSet
+//
+//  Created by Luqman Fauzi on 20/08/2017.
+//  Copyright © 2017 The App Business. All rights reserved.
+//
+
+import UIKit
+import TABObserverSet
+
+extension UIAlertController {
+
+    typealias Grocery = (String, Int)
+
+    func presentGroceryInput<T: UIViewController>(_ target: T, observer: ObserverSet<Grocery>) {
```

Comment:
I see, that's right. I was typealiasing it because on this extension class I intended to add `ObserverSet` functionalities into UIAlertController by passing `ObserverSet<Grocery>` type directly in the parameter instead of passing `ObserverSet<(Int, String)>` type or even creating a model class. 

And also I didn't pass the values into different parameters is because this function takes the `ObserverSet` param and it will broadcast the values (Int & String) through `notify()` method in the `UIAlertAction` completion as the user hit `Add` button, where this `ObserverSet` instance has been observed by the `TupleViewController` class before.